docs/components/concepts/backups.md

```diff
@@ -22,16 +22,30 @@ Exercise caution when deleting clusters to avoid unintended loss of backups.
 
 Your cluster generation needs to be greater or equal to `8.2.4` to support backups.
 
-## Rate limits
+### Rate limits
 
-Each cluster has a limit of three backups. To ensure system stability, backup operations are subject to rate limits. Specifically, you can perform a backup operation every five hours. However, it is possible to delete a backup and create a new one before the 5-hour rate limit expires.
+Each cluster has a limit of three manual backups. To ensure system stability, backup operations are subject to rate limits. Specifically, you can perform a backup operation every five hours. However, it is possible to delete a backup and create a new one before the 5-hour rate limit expires.
 
 The system retains the three most recent completed backups per cluster. Failed backup attempts do not count towards the retention count. When a new backup is successful and the retention count is reached, the oldest backup is automatically deleted.
 
 :::note
 If you require more retained backups or more frequent backups, contact your Customer Success Manager to discuss your specific needs.
 :::
 
+## Scheduled backups
+
+Scheduled backups are backups that are created periodically (e.g daily, weekly..), they can be useful to protect the cluster from major data loss in case of an incident.
+
+### Rate limits
+
+Each cluster has a limit of three successful and three failed scheduled backups, which mean a maximum of 6 scheduled backups are retained.
+
+The system retains the three most recent successful/failed backups per cluster. Failed backup are not re-tried, it just waits for the next scheduled backup to run.
+
+:::note
+If you require more retained backups or more frequent backups, contact your Customer Success Manager to discuss your specific needs.
+:::
```

Comment:
This note is exactly the same for both types of backups. We can put it after both sections instead of duplicating it.